examples/language-modeling/requirements.txt

```diff
@@ -15,4 +15,4 @@ auto-gptq
 openpyxl
 wandb
 py-cpuinfo
-
+autoawq
```

Comment:
remove awq from the requirement

Reply:
verified with validation team: remove awq from autoround requirement and leave it in example requirement.

Reply:
remove it from example too, if users need to export to  awq format, check whether awq has been installed, it not, raise error